### PR TITLE
Fix #4397, #4923: Change the closing button description for consistency & gender neutral profile image

### LINKED ISSUE #4397: [A11Y] Content description for close icon is not consistent.
While checking this PR https://github.com/oppia/oppia-android/pull/4274/files I noticed two things:

1. When we enable screen-reader for `Concept Card` the output for close-icon is `Clicking on this will close the concept card, Button`
2. When we enable screen-reader for `Hints & Solution` the output for close-icon is `Navigate Up, Button`

This needs to be consistent across both screens.
https://github.com/oppia/oppia-android/wiki/Accessibility-A11y-Guide

### LINKED ISSUE #4923: Gender-Neutral Profile Image
**Is your feature request related to a problem?**
We would like to suggest a new feature that would make our platform more inclusive for all users. Currently, when creating a profile, there is only a male profile option available, leaving no option for those who identify as female or gender non-conforming.

**Describe the solution you'd like**
To address this issue, we suggest the introduction of a default gender-neutral profile image that users can choose when creating their profiles. We also suggest having a female profile image option in addition to the existing male profile image option.

**Additional context**

**This is the gender neutral option we propose:**

Screen Shot 2023-03-25 at 9 42 24 PM

**Before:**

**After:**

### PR DESCRIPTION
**Pull request by XichengSpencer**
Fixes #4397: The closing button on the nav bar of concept card activity had inconsistent voice output, it is changed to "navigate up" for consistency.
Fixes #4923: "Have outline person as default gender-neutral image"
 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only
If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes

Screenshot 2023-05-09 120944
Screenshot 2023-05-09 120920
Screenshot 2023-05-09 120932
Screenshot 2023-05-09 134648
Screenshot 2023-05-09 134632

- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide]

Screenshot 2023-05-09 135514
Screenshot 2023-05-09 135209
Screenshot 2023-05-09 135235
Screenshot 2023-05-09 135319

### CODE REVIEW

**Comment by adhiamboperes** [comment]
Hi @XichengSpencer. Thanks for your PR. 

Can you please add a recording in the description that shows the fix is working?

Also counter-check the essential checklist and ensure your PR meets all that are applicable.

**Comment by XichengSpencer** [comment]
Here is the recording

https://user-images.githubusercontent.com/74568012/233457088-8d26b9e0-c05b-485f-8e6c-1252f76871cd.mp4
I checked all the requirements. The title now has contained the fix and issue number, and even though I cannot assign the reviewer but it appears it has automatically assigned one here. Let me know if there are any questions.

**Review by adhiamboperes — COMMENTED** [review]
This looks correct.
@BenHenning, do we generally add automated tests that verify content descriptions?

File: app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt:60]: I think in order to achieve the goal of consistency that #4397 outlines, we probably should be generalizing this string by renaming it to be something like "dismiss_dialog_fragment_icon_description" and ensure that navigate_up is only used in these cases.

File: app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt
Comment (BenHenning) [inline app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt:60]: Since this string no longer seems needed, we should remove it.

**Review by BenHenning — CHANGES_REQUESTED** [review]
Thanks @XichengSpencer!

> This looks correct. @BenHenning, do we generally add automated tests that verify content dsecriptions?

@adhiamboperes @XichengSpencer Yep, we should add a test for situations that can be tested (and content descriptions generally can be). Please do add a test to verify this change @XichengSpencer.

File: app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt
Comment (XichengSpencer) [reply app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt:60]: Hi @BenHenning,
I understand that names like "dismiss_dialog_fragment_icon_description" will have a clear description for the purpose of the string. But when I looked into the usage of "R.string.navigate_up", it was used in both fragments and activities that serve different purposes. On the other hand, if we create different words to describe the function of navigate up buttons while having the same string value of "navigate up" seems redundant to me.  Overall, I think the name "navigate up" is a good generalization to loose coupling to all functions while maintaining simplicity. It is just like having the "go back to the previous page" button on your browser that will serve the same function regardless of what page you are on.

Hope this makes sense and looking forward to your thought on this.

File: app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt
Comment (XichengSpencer) [reply app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt:60]: Sure

**Comment by XichengSpencer** [comment]
For #4923 here are the result screenshots

**Comment by adhiamboperes** [comment]
Hey @XichengSpencer. I would advise that you open one PR per issue that you are working on, for now.

**Comment by XichengSpencer** [comment]
> Hey @XichengSpencer. I would advise that you open one PR per issue that you are working on, for now.

Sorry, I forgot to create a new branch for this one. I will definitely keep it in mind for the next time. But what should I do for this one? The second issue is just a minor resource change here.

**Comment by adhiamboperes** [comment]
> > Hey @XichengSpencer. I would advise that you open one PR per issue that you are working on, for now.
> 
> Sorry, I forgot to create a new branch for this one. I will definitely keep it in mind for the next time. But what should I do for this one? The second issue is just a minor resource change here.

We can keep this one this time, although in future only close multiple issues in one PR if they are thematically related.

**Review by adhiamboperes — APPROVED** [review]
Thanks @XichengSpencer. Your changes look good to me.

Can you also fill out the screenshots section of the PR description and then assign your pr over to Ben by tagging him and mentioning "PTAL"?

**Comment by XichengSpencer** [comment]
> Thanks @XichengSpencer. Your changes look good to me.
> 
> Can you also fill out the screenshots section of the PR description and then assign your pr over to Ben by tagging him and mentioning "PTAL"?

Done. Thank you so much for the review!

**Comment by XichengSpencer** [comment]
Hi @BenHenning 
Changes have been made, PTAL. Thank you!

**Review by BenHenning — COMMENTED** [review]
Thanks @XichengSpencer! This LGTM from a code perspective.

@seanlip could you or someone from UX weigh in on the icon change before we finalize this & merge?

**Comment by seanlip** [comment]
Hm, I'm a bit dubious about the new icon. I'm a bit unsure about using the outline version instead of the filled version.

Let me ask the design team for some feedback -- stay tuned. I'll send an email.

**Comment by seanlip** [comment]
Assigning to @kaiyuxu (Android design lead) for feedback.

File: app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt
Comment (BenHenning) [reply app/src/main/java/org/oppia/android/app/topic/conceptcard/ConceptCardFragmentPresenter.kt:60]: We could probably revisit this once we finish organizing the string locations, so I don't think more is needed at this point. Thanks!

**Review by BenHenning — COMMENTED** [review]
Thanks @XichengSpencer. Waiting on the design team's feedback before approving in case there are any additional changes needed.

**Comment by seanlip** [comment]
@BenHenning Confirming this is all fine from a design and product perspective (see email) -- let's get this in. Thanks!

**Review by BenHenning — APPROVED** [review]
Thanks @seanlip for confirming, and thanks @XichengSpencer for wrapping this up! LGTM.

**Comment by BenHenning** [comment]
Updating the PR to the latest develop & enabling auto-merge.

**Pull request merged**
